types/index.d.ts

```diff
@@ -58,6 +58,20 @@ declare namespace inject {
     Request?: object,
   }
 
+  /**
+   * https://github.com/nfriedly/set-cookie-parser/blob/3eab8b7d5d12c8ed87832532861c1a35520cf5b3/lib/set-cookie.js#L41
+   */
+  interface Cookie {
+    name: string;
+    value: string;
+    expires?: Date;
+    maxAge?: number;
+    secure?: boolean;
+    httpOnly?: boolean;
+    sameSite?: string;
+    [name: string]: unknown
```

Comment:
```suggestion
    [key: string]: string
```

It is not unknown, but always string ;)

Reply:
Yes not it can't be string here

Reply:
`string` conflict with known non-string property like `httpOnly`

Reply:
so it will be `unknown` or `any` or `string|boolean|Date`

Reply:
when you get property with index like `cookies['httpOnly']`, ts will return type `unknown` instead known `boolean`